src/playground/App.js

```diff
@@ -72,20 +103,9 @@ const App = () => {
     const { text: urlText, options: urlOptions } = getUrlState();
     const [text, setText] = useState(urlText || storedText || "/* eslint quotes: [\"error\", \"double\"] */\nconst a = 'b';");
     const [fix, setFix] = useState(false);
-    const [options, setOptions] = useState(fillOptionsDefaults(urlText ? urlOptions || {} : storedOptions));
+    const [options, setOptions] = useState(fillOptionsDefaults(urlText ? urlOptions || {} : convertLegacyOptionsToFlatConfig(storedOptions)));
 
     const lint = () => {
-        try {
-            const validator = new Legacy.ConfigValidator({ builtInRules: rules });
-
-            validator.validate(options);
-        } catch (error) {
-            return {
-                messages: [],
-                output: text,
-                validationError: error
-            };
-        }
```

Comment:
Good Catch! Ideally, it would be great if we could set an `error.code = "ESLINT_VALIDATION_ERROR` or a specific exit code in case of validation errors. Right now I'm evaluating it based on the presence of`key "rules"` in error message.